### PR TITLE
Update merge_sort.py

### PR DESCRIPTION
**Pull request by sidisonline**
just learning new thing

### CODE REVIEW

File: merge_sort.py [hunk 1]
```diff
@@ -41,3 +41,8 @@ def merge_two_sorted_lists(a,b,arr):
 arr=[12,45,78,34,99,11,55]
 merge_sort(arr)
 print(arr)
+
```
Comment (sidisonline) [inline merge_sort.py:44]: just a new input/output

**Review by sidisonline — COMMENTED** [review]
soooo

File: merge_sort.py (hunk 1 above)
Comment (jainsiddharth99) [reply merge_sort.py:44]: ohhk

Comment (jainsiddharth99) [reply merge_sort.py:44]: a little more needs to be done

**Review by jainsiddharth99 — APPROVED** [review]

**Comment by jainsiddharth99** [comment]
add space in arr1

**Pull request closed without merging**